integration-cli/docker_cli_daemon_test.go

```diff
@@ -2489,12 +2489,13 @@ func (s *DockerDaemonSuite) TestDaemonWithUserlandProxyPath(c *testing.T) {
 	out, err = s.d.Cmd("run", "-p", "5000:5000", "busybox:latest", "true")
 	assert.NilError(c, err, out)
 
+	// not an absolute path
+	err = s.d.RestartWithError("--userland-proxy-path", "docker-proxy")
+	assert.ErrorContains(c, err, "invalid userland-proxy-path: must be an absolute path: docker-proxy")
```

Comment:
Ah, dang; error only returns `exit 1`. Probably need some other way to get the output;

```
=== FAIL: amd64.integration-cli TestDockerDaemonSuite/TestDaemonWithUserlandProxyPath (2.14s)
    docker_cli_daemon_test.go:2494: assertion failed: expected error to contain "invalid userland-proxy-path: must be an absolute path: docker-proxy", got "[dde8a51955c4d] daemon exited during startup: exit status 1"
        exit status 1
        [dde8a51955c4d] daemon exited during startup
        github.com/docker/docker/testutil/daemon.(*Daemon).StartWithLogFile
        	/go/src/github.com/docker/docker/testutil/daemon/daemon.go:556
        github.com/docker/docker/testutil/daemon.(*Daemon).StartWithError
        	/go/src/github.com/docker/docker/testutil/daemon/daemon.go:417
        github.com/docker/docker/testutil/daemon.(*Daemon).RestartWithError
        	/go/src/github.com/docker/docker/testutil/daemon/daemon.go:750
        github.com/docker/docker/integration-cli.(*DockerDaemonSuite).TestDaemonWithUserlandProxyPath
        	/go/src/github.com/docker/docker/integration-cli/docker_cli_daemon_test.go:2493
        reflect.Value.call
        	/usr/local/go/src/reflect/value.go:596
        reflect.Value.Call
        	/usr/local/go/src/reflect/value.go:380
        github.com/docker/docker/internal/test/suite.Run.func2
        	/go/src/github.com/docker/docker/internal/test/suite/suite.go:69
        testing.tRunner
        	/usr/local/go/src/testing/testing.go:1595
        runtime.goexit
        	/usr/local/go/src/runtime/asm_amd64.s:1650
    check_test.go:589: [dde8a51955c4d] daemon is not started
    --- FAIL: TestDockerDaemonSuite/TestDaemonWithUserlandProxyPath (2.14s)
```